Tests/Functional/Command/DebugGatewayCommandTest.php

```diff
@@ -113,7 +113,7 @@ protected function executeConsole(Command $command, array $arguments = [], array
         return $commandTester->getDisplay();
     }
 
-    protected function getInputStream($input)
+    protected function getInputStream($input): bool
```

Comment:
Is this return type correct?  I think this will actually return `resource|false`, but `resource` can't be used as a type, so the return can be left empty, and a dockblock added with `@return resource|false`

Reply:
I agree. PhpStorm still wants to add bool. So i added a @noinspection PhpMissingReturnTypeInspection to silence this.